actors/market/src/policy.rs

```diff
@@ -10,14 +10,19 @@ use fvm_shared::clock::ChainEpoch;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::piece::PaddedPieceSize;
 use fvm_shared::sector::StoragePower;
-use fvm_shared::TOTAL_FILECOIN;
+use lazy_static::lazy_static;
 use num_traits::Zero;
 
 pub mod detail {
     /// Maximum length of a deal label.
     pub const DEAL_MAX_LABEL_SIZE: usize = 256;
 }
 
+lazy_static! {
+    /// Total Filecoin available to the network.
```

Comment:
Commentary: The _assumed_ total filecoin. We should really make this a syscall.

Reply:
TBH, this constant should just go away. It's used to specify collateral bounds, which is kind of unnecessary IMO (although I guess the idea is to limit the size of state?).